src/docs/product/reference/search/searchable-properties/index.mdx

```diff
@@ -14,7 +14,7 @@ Sentry's searchable properties fall into one of four categories:
 
 - [Issue properties](/product/sentry-basics/search/searchable-properties/issues/)
 - [Event properties](/product/sentry-basics/search/searchable-properties/events/)
-- [Session replay properties](/product/sentry-basics/search/searchable-properties/session-replay/)
+- [Session Replay properties](/product/sentry-basics/search/searchable-properties/session-replay/)
 - [Release properties](/product/sentry-basics/search/searchable-properties/releases/)
```

Comment:
We're missing a User Feedback link in this list:
![Screenshot 2023-11-28 at 1 30 44 PM](https://github.com/getsentry/sentry-docs/assets/77064737/5bd2ae47-ac37-4e18-86f3-8e7717404949)